dbt_server/views.py

```diff
@@ -388,6 +404,44 @@ async def run_operation_async(
 ):
     return task_service.run_operation_async(background_tasks, db, args)
 
+@app.post("/async/dbt/")
+async def dbt_entry(
+    # background_tasks: BackgroundTasks,
+    args: List[str]= Query(None),
+    # request: Request,
+    # commons: list = Depends(common_parameters),
+    # args: list,
+    # response_model=schemas.Task,
+    # db: Session = Depends(crud.get_db),
+):  
+    # example request:
+    # http://127.0.0.1:8580/async/dbt/?args=--printer-width&args=100&args=run&args=--project-dir&args=/Users/chenyuli/git/python-models-test-project-small
+    
+    set_invocation_args(args)
+    # create invocation ctx, this will take care of validation of args
+    try:
+        ctx = DBTContext(args)
```

Comment:
In the future we plan to add more function to DBTContext, you can imagine we would be able to do
``` python
profile = new_core_api.load_profile(profile_json)

ctx.set_profile(profile)
ctx.set_project(project) # project and profile are being used to construct RuntimeConfig
ctx.set_manifest(manifest)
ctx.set_logger(custom_logger)
```